app/App.js

```diff
@@ -34,19 +34,6 @@ class App extends Component<{
   actions: ActionsMap,
   history: Object,
 }> {
-
-  mobxDevToolsInstanceIfDevEnv(): ?React$Element<any> {
```

Comment:
Explanation: I remove mobx-react-devtools because the package has been deprecated and replaced by a browser extension